app/models/journaled/writer.rb

```diff
@@ -28,12 +28,13 @@ def journal!
     validate!
     Journaled::DeliveryJob
       .set(journaled_enqueue_opts.reverse_merge(priority: Journaled.job_priority))
-      .perform_later(delivery_perform_args)
+      .perform_later(**delivery_perform_args)
```

Comment:
This technically alters the job handler - instead of a single hashy argument, these will now be passed as keyword arguments. But this only really impacts ruby 3.0+, and I think that anyone attempting to use this gem on Ruby 3.0 would've quickly found that jobs would fail to execute due to the hash/kwarg mismatch between this call and the class's perform method.

So, since I did not alter the `Journaled::ChangeJob`'s contract, any already-enqueued jobs should still work on Ruby 2.7 and below -- we just need to make sure that when we upgrade to Ruby 3.0, there are no jobs in the queue from before this change.